es-ds-components/components/es-slider.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import Slider from 'primevue/slider';
+
+defineProps({
+    min: {
+        type: Number,
+        required: true,
+        default: 0,
+    },
+    max: {
```

Comment:
I don't think it's a good idea to change the props that this component is expecting.
https://github.com/search?q=org%3AEnergySage%20esslider&type=code